package.json

```diff
@@ -80,27 +80,27 @@
 		"sodium-native": "LiskHQ/sodium-native#dd0319f",
 		"strftime": "=0.10.0",
 		"swagger-node-runner": "=0.7.3",
-		"sway": "=1.0.0",
+		"sway": "=2.0.5",
 		"valid-url": "=1.0.9",
 		"wamp-socket-cluster": "=2.0.0-beta.4",
 		"z-schema": "=3.18.2"
 	},
 	"devDependencies": {
-		"bitcore-mnemonic": "=1.2.5",
+		"bitcore-mnemonic": "=1.5.0",
```

Comment:
This package is not being used in the repository, it's better to verify one more time and remove the packages which are not being used.

Reply:
I'm creating new issues for a more general cleanup/update of packages (https://github.com/LiskHQ/lisk/issues/2499 and https://github.com/LiskHQ/lisk/issues/2498) would you prefer for this change to happen in this PR or in the new issues? Also I couldn't find usages of chai-bignumber

Reply:
Removed bitcore-mnemonic and chai-bignumber from devDependencies as they were not being used.

Reply:
@ManuGowda I had to re-add the dependencies as the build was breaking. Will investigate further.